public/v4/apps/adguard-home.yml

```diff
@@ -0,0 +1,42 @@
+captainVersion: 4
+services:
+    # AdGuard
+    $$cap_appname:
+        image: adguard/adguardhome:$$cap_adguard_version
+        restart: always
+        ports:
+            - 53:53
+            - 67:67
+            - 68:68
+            - 8081:80
+            - 4433:443
```

Comment:
Are these all required? Wouldn't nginx regular proxy be enough for 80/443?

Reply:
yea let me fix that